src/js/tabs/cli.js

```diff
@@ -210,7 +211,13 @@ cli.initialize = function (callback) {
         $("a.save").on("click", function () {
             const filename = generateFilename("cli", "txt");
 
-            saveFile(filename, self.outputHistory);
+            let content = self.outputHistory;
+
+            if (self.lastSupportId) {
+                content = `Support ID: ${self.lastSupportId}\n\n${content}`;
```

Comment:
```suggestion
                content = `# Support ID: ${self.lastSupportId}\n\n${content}`;
```
hashmark for copy/paste exclusion